### PR TITLE
Add Share Button

### PR DESCRIPTION
**Pull request by tikhsuP**
This PR adds a SHARE button in the extension UI that enables the user to copy the extension link to the clipboard. 

Share button

### CODE REVIEW

File: scripts/popup.js [hunk 1]
```diff
@@ -300,6 +300,15 @@ function settings() {
   $('#setting-page').show()
 }
 
+document.getElementById('webstore-url').style.display = 'none';
+function share(webstoreURL) {
+  let $temp = $('<input>')
+  $('body').append($temp)
+  $temp.val($(webstoreURL).text()).select()
+  document.execCommand('copy')
```
Comment (anishsarangi) [inline scripts/popup.js:308]: Have you tested it in firefox?

File: scripts/popup.js [hunk 2]
```diff
@@ -300,6 +300,15 @@ function settings() {
...
+  $temp.remove()
```
Comment (anishsarangi) [inline scripts/popup.js:309]: Why are you removing it?

File: scripts/popup.js (hunk 1 above)
Comment (tikhsuP) [reply scripts/popup.js:308]: It doesn't works for firefox. I am searching for a fix, will update when I find it.

File: scripts/popup.js [hunk 3]
```diff
@@ -300,6 +300,23 @@ function settings() {
...
 }
 
+function share(){
+  const webstoreURL = 'https://chrome.google.com/webstore/detail/wayback-machine/fpnmgdkabkmnadcjpehmlllkndpkmiak'
+  navigator.clipboard.writeText(webstoreURL).then(function() {
+    let toastMsg = document.getElementById('toast-success')
+    toastMsg.classList.add('show')
+    setTimeout(function(){
+      toastMsg.classList.remove('show')
+    }, 1000)
```
[the remark this answers is not in the document]
Comment (tikhsuP) [reply scripts/popup.js:309]: Made the required changes.

File: scripts/popup.js [hunk 4]
```diff
@@ -300,6 +300,27 @@ function settings() {
   $('#setting-page').show()
 }
 
+function share(){
+  const webstoreURL = 'https://chrome.google.com/webstore/detail/wayback-machine/fpnmgdkabkmnadcjpehmlllkndpkmiak'
+  function showToast(toastMsg){
```
Comment (anishsarangi) [inline scripts/popup.js:305]: Why you have created a function here?

File: scripts/popup.js [hunk 5]
```diff
@@ -300,6 +300,27 @@ function settings() {
...
+    toastMsg.classList.add('show')
+    setTimeout(function(){
+      toastMsg.classList.remove('show')
+    }, 1000)
+  }
+  navigator.clipboard.writeText(webstoreURL).then(function() {
+    const toastMsg = document.getElementsByClassName('toast')
```
Comment (anishsarangi) [inline scripts/popup.js:312]: Move this line out side of this scope and declare it there once and remove line no-307

File: scripts/popup.js (hunk 4 above)
Comment (tikhsuP) [reply scripts/popup.js:305]: To reuse the code to show the toast.

Comment (anishsarangi) [reply scripts/popup.js:305]: Define it out side the share function then !Thank you

File: scripts/popup.js (hunk 5 above)
Comment (tikhsuP) [reply scripts/popup.js:312]: Moved it, but still confused about line 307. Should I remove the timeout function?

**Comment by anishsarangi** [comment]
Good to merge. Thank you !

**Comment by cgorringe** [comment]
Can we hold off merging this? I actually don't agree that this icon should be added to the popup, but the link should be provided on the About page instead.

**Comment by tikhsuP** [comment]
Sure, as you guys agree on. 🙂

**Comment by anishsarangi** [comment]
@tikhsuP Let's hold this PR for some time. As @cgorringe suggested, we can add in the about page. Let's work on that

**Comment by tikhsuP** [comment]
@cgorringe is working on the About page #504

**Pull request closed without merging**